### PR TITLE
update searchdav to 0.3.1

### PR DESCRIPTION
**Pull request by icewind1991**

### CODE REVIEW

File: composer/autoload_classmap.php [hunk 1]
```diff
@@ -615,6 +615,20 @@
...
+    'Icewind\\Streams\\Tests\\DirectoryWrapperNull' => $vendorDir . '/icewind/streams/tests/DirectoryWrapper.php',
+    'Icewind\\Streams\\Tests\\FailWrapper' => $vendorDir . '/icewind/streams/tests/RetryWrapper.php',
+    'Icewind\\Streams\\Tests\\IteratorDirectory' => $vendorDir . '/icewind/streams/tests/IteratorDirectory.php',
+    'Icewind\\Streams\\Tests\\NullWrapper' => $vendorDir . '/icewind/streams/tests/NullWrapper.php',
+    'Icewind\\Streams\\Tests\\PartialWrapper' => $vendorDir . '/icewind/streams/tests/RetryWrapper.php',
+    'Icewind\\Streams\\Tests\\PathWrapper' => $vendorDir . '/icewind/streams/tests/PathWrapper.php',
+    'Icewind\\Streams\\Tests\\RetryWrapper' => $vendorDir . '/icewind/streams/tests/RetryWrapper.php',
+    'Icewind\\Streams\\Tests\\SeekableWrapper' => $vendorDir . '/icewind/streams/tests/SeekableWrapper.php',
+    'Icewind\\Streams\\Tests\\UrlCallBack' => $vendorDir . '/icewind/streams/tests/UrlCallBack.php',
+    'Icewind\\Streams\\Tests\\Wrapper' => $vendorDir . '/icewind/streams/tests/Wrapper.php',
```
Comment (icewind1991) [inline composer/autoload_classmap.php]: They are in gitignore, but since I have the files locally composer helpfully adds them to the classmap

Comment (MorrisJobke) [inline composer/autoload_classmap.php]: Then remove them also locally and update that file ;)

Comment (icewind1991) [inline composer/autoload_classmap.php]: done

**Comment by MorrisJobke** [comment]
Sign off missing ... then please merge this and update the server PR

**Comment by icewind1991** [comment]
> Sign off missing ... then please merge this and update the server PR

Done

**Pull request merged**